### DISCUSSION TITLE
How to find functions in unity 3D

### DISCUSSION THREAD

**Discussion by Edblue3** [microsoft/vscode-discussions#589] [Extension Development QnA]
How do I find functions in unity 3D when there are so many I only know one function name applydamage. I posted a forum question on unity community and didn't receive any responses. Also when I contact unity they only help with non-technical support.

**Comment by gjsjohnmurray** [comment]
This discussion forum is focused on developers of extensions for VS Code. Please see https://github.com/microsoft/vscode-discussions/discussions/5 for advice about other places to interact. In your case I think StackOverflow would be the right choice.

**Comment by olguzzar** [comment]
Closing this discussion as it is off-topic for the extension development forum.
